### DISCUSSION TITLE
Update styling on a single point/polygon

### DISCUSSION THREAD

**Discussion by sunyonglincanada** [heremaps/harp.gl#2102] [Q&A]
Hi there,

I was trying to change one of points/polygon from geojson data. But I did not find a proper way to update it. Take the demo for an example - https://imgur.com/a/jjEYlHX, I want to update one specific point color when clicking on it. Is that doable?

I was trying to add the same data source to override the old data with a different color. But this method does not work on a single point. Here is my code - https://github.com/sunyonglincanada/harp.gl-GLTF

Can anyone take a look to see if there is a proper way to do this? Thanks!!

**Comment by nzjony** [comment]
@atomicsulfate , do you know? I guess it could have to do with picking the point and assigning some dynamic property?
